themes/default/gulpfile.js

```diff
@@ -221,6 +236,8 @@ gulp.task('make-js', ['make-js-components', 'make-js-npm'], function() {
 		.pipe(gulp.dest('js'))
 });
 
+gulp.task('build', ['pure', 'make-css', 'cms-css', 'make-js', 'svgo']);
```

Comment:
I'd definitely add json into the build, although not add svgo to watch as it would slow it down hugely